### ISSUE TITLE
Build errors when using CocoaLumberjack v2.0.0

### ISSUE THREAD

**Issue by master-nevi** [swisspol/GCDWebServer#151]
Currently I get the following build error:
```
GCDWebServer.m:77:28: Use of undeclared identifier 'LOG_LEVEL_DEBUG'
```
The build errors are reproducible by creating a brand new Swift project with a Podfile that installs the following pods using CocoaPods v0.36.0.rc.1:
```ruby
pod 'GCDWebServer'
pod 'CocoaLumberjack', '~> 2.0.0-rc2'
```

I saw that CocoaLumberjack had a [laundry list of items to migrate to v2.0.0](https://github.com/CocoaLumberjack/CocoaLumberjack#migrating-to-2x), one of which was to replace LOG_LEVEL_* macros with DDLogLevel enum values, so I applied this hoping for an easy fix:
```objective-c
#if DEBUG
int GCDWebServerLogLevel = DDLogLevelDebug;
#else
int GCDWebServerLogLevel = DDLogLevelInfo;
#endif
```
But alas, I then get a different set of errors:
```
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_DDLog", referenced from:
      objc-class-ref in GCDWebServer.o
      objc-class-ref in GCDWebServerConnection.o
      objc-class-ref in GCDWebServerFileResponse.o
      objc-class-ref in GCDWebServerFunctions.o
      objc-class-ref in GCDWebServerMultiPartFormRequest.o
      objc-class-ref in GCDWebServerRequest.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
I usually open up Github issues with a workaround or a possible fix, but at this point I don't know enough about how GCDWebServer works to continue.

**Comment by swisspol** [comment]
The linking issue seems like a CocoaLumberJack or CocoaPod issue, a GCDWebServer: if you search the GCDWebServer source you will notice it doesn't use the `DDLog` class anywhere. It must come from one of the CocoaLumberJack macros and for some reason the CocoaLumberJack don't get added by CocoaPods or something.

**Comment by klaaspieter** [comment]
Another problem is that the log level names were changed from `LOG_LEVEL_*` to `DDLogLevel*`. [These lines](https://github.com/swisspol/GCDWebServer/blob/master/GCDWebServer/Core/GCDWebServer.m#L77-L79) hardcode the log levels to those used in CocoaLumberjack < 2.

**Comment by swisspol** [comment]
Roll back to a pre-2.0 version of CocoaLumberJack or patch GCDWebServer `Private.h` file to not forward logging to CocoaLumberJack or use completely custom logging which could even be a no-op:
https://github.com/swisspol/GCDWebServer/blob/master/GCDWebServer/Core/GCDWebServer.h#L481

**Comment by swisspol** [comment]
Should be fixed now: 594497d2348362e64a5dc3a376b11aedb8ebc81a

**Comment by cipherCOM** [comment]
See https://github.com/swisspol/GCDWebServer/issues/214 for a solution to the _DDLog symbol not found problem.
